OpenSim/Simulation/MarkersReference.h

```diff
@@ -123,24 +90,18 @@ OpenSim_DECLARE_CONCRETE_OBJECT(MarkersReference, Reference_<SimTK::Vec3>);
     //--------------------------------------------------------------------------
     MarkersReference();
 
-    // Convenience load markers from a file
-    MarkersReference(const std::string filename, Units modelUnits=Units(Units::Meters));
-
-    MarkersReference(MarkerData& aMarkerData, const Set<MarkerWeight>* aMarkerWeightSet=NULL);
+    /** Convenience load markers from a file */
+    MarkersReference(const std::string& markerFileName,
+                     Units modelUnits=Units(Units::Meters));
+    /** Form a Reference from Marker data (taking ownership) and weights */
+    MarkersReference(MarkerData* markerData,
+        const Set<MarkerWeight>* markerWeightSet=nullptr);
```

Comment:
This constructor is a little confusing because `MarkersReference` takes ownership of `markerData` but makes a copy of `markerWeightSet`. Can you make this more clear in the doxygen?